File: Python/banners.py
import requests

# Set up the authentication
api_key = "YOUR_API_KEY_HERE"

# Set up the request headers
headers = {
    "Authorization": "Bearer " + api_key,
    "Content-Type": "application/json"
}

# Make the request to get the store data
store_response = requests.get("https://fortniteapi.com/v2/shop/br", headers=headers)

# Parse the response to get the available banners
store_data = store_response.json()
available_banners = []
for item in store_data['data']['featured']:
    if item['type'] == 'Banner':
        available_banners.append(item)

# Buy the first banner in the available list
if len(available_banners) > 0:
    banner_to_buy = available_banners[0]
    buy_data = {
        "item": {
            "item_id": banner_to_buy['itemId'],
            "item_category": banner_to_buy['itemCategory'],
            "item_rarity": banner_to_buy['itemRarity'],
            "item_type": banner_to_buy['itemType']
        },
        "currency": "MtxCurrency",
        "quantity": 1
    }
    buy_response = requests.post("https://fortniteapi.com/v2/shop/br/buy", headers=headers, json=buy_data)
    if buy_response.status_code == 200:
        print("Successfully purchased banner:", banner_to_buy['name'])
    else:
        print("Error purchasing banner:", buy_response.status_code, buy_response.text)
else:
    print("No banners available to purchase")

File: JS/banners.js
const fetch = require('node-fetch');

// Set up the authentication
const api_key = "YOUR_API_KEY_HERE";

// Set up the request headers
const headers = {
  "Authorization": "Bearer " + api_key,
  "Content-Type": "application/json"
};

// Make the request to get the store data
fetch("https://fortniteapi.com/v2/shop/br", { headers })
  .then(response => response.json())
  .then(store_data => {
    const available_banners = store_data.data.featured.filter(item => item.type === 'Banner');

    // Buy the first banner in the available list
    if (available_banners.length > 0) {
      const banner_to_buy = available_banners[0];
      const buy_data = {
        "item": {
          "item_id": banner_to_buy.itemId,
          "item_category": banner_to_buy.itemCategory,
          "item_rarity": banner_to_buy.itemRarity,
          "item_type": banner_to_buy.itemType
        },
        "currency": "MtxCurrency",
        "quantity": 1
      };

      fetch("https://fortniteapi.com/v2/shop/br/buy", {
        method: 'POST',
        headers,
        body: JSON.stringify(buy_data)
      })
      .then(buy_response => {
        if (buy_response.status === 200) {
          console.log("Successfully purchased banner:", banner_to_buy.name);
        } else {
          console.log("Error purchasing banner:", buy_response.status, buy_response.statusText);
        }
      });
    } else {
      console.log("No banners available to purchase");
    }
  });
